lib/config.js

```diff
@@ -130,6 +130,13 @@ const Config = {
         "If enabled, all files of the same grammar will share a single global kernel (requires atom restart)",
       type: "boolean",
       default: false
+    },
+    importNotebookURI: {
+      title: "Enable Notebook Auto-import",
+      description:
+        "If enabled, opening a Jupyter Notebook file in Atom (with extension `.ipynb`) will instead import the file's source into a new tab.",
```

Comment:
What about linking to the docs and mentioning that Hydrogen has to be active. Something like:

"If enabled, opening a file with extension `.ipynb` will [import the notebook](https://nteract.gitbooks.io/hydrogen/docs/Usage/NotebookFiles.html#notebook-import) file's source into a new tab. If disabled, or if the Hydrogen package is not activated, the raw file will open in Atom as normal."

Reply:
You're right. I assume that we don't activate Hydrogen at Atom startup to save time? I was hoping the URI would trigger a Hydrogen activation command, but it doesn't look like it does

Reply:
I think there is a way to do something minor with a package like adding an opener without activating the package. We can look into it and add in another PR if you want that behavior, I'd ask in the atom slack or http://discuss.atom.io.